frontend/src/main/java/de/iav/frontend/controller/AddMetrologyViewController.java

```diff
@@ -0,0 +1,100 @@
+package de.iav.frontend.controller;
+
+import de.iav.frontend.model.Metrology;
+import de.iav.frontend.service.IdService;
+import de.iav.frontend.service.MetrologyService;
+import de.iav.frontend.service.SceneSwitchService;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.control.DatePicker;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.Date;
+
+public class AddMetrologyViewController {
+
+    @FXML
+    private TextField TF_INVENTORY;
+    @FXML
+    private TextField TF_MANUFACTURER;
+    @FXML
+    private TextField TF_TYPE;
+    @FXML
+    private Label LF_ERROR;
+    @FXML
+    private DatePicker DP_MAINTENANCE;
+    @FXML
+    private DatePicker DP_CALIBRATION;
```

Comment:
Diese Variablennamen sollten dem regulären camelCase folgen.
Unterstriche sind in dem Sinne auch nicht Teil der Konvention.

Optimalerweise würde man auch textField nicht abkürzen.